Modules/SubModule_ASL/xASL_wrp_Quantify.m

```diff
@@ -456,5 +466,8 @@ function xASL_wrp_Quantify(x, PWI_Path, OutputPath, M0Path, SliceGradientPath)
     end
     
     xASL_spm_deformations(x, {x.P.Path_CBF}, {x.P.Pop_Path_CBF}, [], [], AffineTransfPath, x.P.Path_y_ASL);
+	if xASL_exist(x.P.Path_TT,'file')
+		xASL_spm_deformations(x, {x.P.Path_TT}, {x.P.Pop_Path_TT}, [], [], AffineTransfPath, x.P.Path_y_ASL);
```

Comment:
Should it be `Path_ATT` instead? `Path_TT` I used in the past for the FEAST output, we could clean this up in the future

Reply:
Up to you, I have used the previous notation. But I'm fine to rename this. Do you want to use ATT for both FEAST and non-FEAST?

Reply:
Perhaps use something that works for now, and soon update to BIDS terminology?

Reply:
Yes. Can you resolve?

Reply:
I apologise for opening this again, but what is the difference? Both produce ATT maps?

Reply:
No worries. There's no difference. It was just called TT previously, so while ATT is better, we will keep the initial name. And we also don't want to have both TT and ATT as this would be confusing.

There's of course a difference in the algorithms - one works with multi-PLD, the other with single-PLD and crusher ON/OFF. 

If that explains it, you can resolve this one.

Reply:
> @jan-petr, @BeatrizPadrela, @MDijsselhof & @HenkMutsaerts: Let's merge as is and test in release branch. Easier and better IMHO 👍

We should first agree on the structure, then test this branch, and then test the release branch. But we can fuse the last two to speed things up, for now. And we could move things to a new issue.

But the clarity in `xASL_wrp_Realign` what is done when, is something I would like to implement now, saves time testing and discussing.